pkg/client/cache/cache.go

```diff
@@ -19,6 +19,6 @@ package cache
 import "github.com/codenotary/immudb/pkg/api/schema"
 
 type Cache interface {
-	Get(serverUuid string) (*schema.Root, error)
-	Set(root *schema.Root, serverUuid string) error
+	Get() (*schema.Root, error)
```

Comment:
this has not to be done in order to keep compatibility with https://github.com/codenotary/immudb/commit/2293ba67bf020b8c2d2851e2ec91942a492a86d3